string port     # MSの接続されているUSBポート
uint8 msid      # 自身のID
uint8 type      # 未定義:0 Noid:1 Lifter:2 Mover:3 
